Documentation/markdown/SetupHelp.md

```diff
@@ -24,7 +24,11 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
 
 		* **HINT:** There are components, which Qt Online Installer selects as default options. You can uncheck these components for saving computer memory.
 
-4. **Boost 1_65_1** is installed (find [here](https://sourceforge.net/projects/boost/files/boost-binaries/1.65.1/boost_1_65_1-msvc-14.1-64.exe/download)) and a new environment variable called Boost_INCLUDE_DIR is created. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. C:\local\boost_1_65_1\).
+4. **Boost 1_65_1** is installed (find [here](https://sourceforge.net/projects/boost/files/boost-binaries/1.65.1/boost_1_65_1-msvc-14.1-64.exe/download)).
+
+	* Create a folder named thirdparty in C:\ and install **Boost 1_65_1** to C:\thirdparty\vs2015\x64\boost_1_65_1.
```

Comment:
```suggestion
	* Create a folder named thirdparty in `C:\` and install **Boost 1_65_1** to `C:\thirdparty\vs2015\x64\boost_1_65_1`.
```
Please use markdown's formating for code snippets.

Reply:
Resolved: 77865a7